target/linux/ath79/dts/qca9563_glinet_gl-ar750s-nor-nand.dts

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/dts-v1/;
+
+#include "qca9563_glinet_gl-ar750s.dtsi"
+
+/ {
+	compatible = "glinet,gl-ar750s-nor-nand", "qca,qca9563";
+};
```

Comment:
This should contain a model (including NOR/NAND suffix).

Reply:
It might be not enough coffee this morning,  but what is the recommended change here? "glinet,gl-ar750s-nor-nand" seems to me to be the DTS name of the device.

Reply:
You kept `model = "GL.iNet GL-AR750S";` in the DTSI.
You should move it the DTS files and use
`model = "GL.iNet GL-AR750S (NOR/NAND)";`
and
`model = "GL.iNet GL-AR750S (NOR)";`
in the respective DTS files.

By that, the model becomes equivalent to the DEVICE_TITLE constructed from DEVICE_VENDOR, _MODEL and _VARIANT ...

Reply:
_(Correction made)_